### ISSUE TITLE
filter_fields' lookups seems to be ignored

### ISSUE THREAD

**Issue by id13** [barseghyanartur/django-elasticsearch-dsl-drf#7]
We I declare something like this

```
filter_fields = {
   'foobar': {
       'field': 'foobar',
       'lookups': [
           LOOKUP1,
           LOOKUP2,
           # etc ..
       ],
   },
}
```

My lookups seem to be ignored, what is used instead is the content of `ALL_LOOKUP_FILTERS_AND_QUERIES` [here](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/constants.py#L268).

It is due to [this line](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/src/django_elasticsearch_dsl_drf/filter_backends/filtering/common.py#L103), maybe it should be `lookups` instead of `filters`.

I can fix it if you confirm it is actually a bug.

**Comment by barseghyanartur** [comment]
@id13:

Well spotted. I've fixed that already. A hot-fix is coming. Thanks!

**Comment by barseghyanartur** [comment]
@id13:

Fixed in [0.3.12](https://pypi.python.org/pypi/django-elasticsearch-dsl-drf/0.3.12).
